armory/data/adversarial_datasets.py

```diff
@@ -830,6 +830,29 @@ def __call__(self, x, labels):
         return boxes, patch_metadata_dict
 
 
+class ClipMOTVideoTrackingLabels(ClipVideoTrackingLabels):
```

Comment:
How/where is `ClipMOTVideoTrackingLabels` used?

```
(armory) lucas@ubu0953:~/Desktop/gard/git/twosixlabs/armory$ grep -r "ClipMOTVideoTrack" .
Binary file ./armory/data/__pycache__/adversarial_datasets.cpython-38.pyc matches
./armory/data/adversarial_datasets.py:class ClipMOTVideoTrackingLabels(ClipVideoTrackingLabels):
(armory) lucas@ubu0953:~/Desktop/gard/git/twosixlabs/armory$ 
```

Reply:
I also think its __init__() is identical to that of the parent class it overrides

Reply:
if `y` has shape `(batch_size, num_detections, 9)`, I'm not seeing how `(y[y[:, :, 0] < self.max_frames]` makes sense as it looks like it's only indexing the first of the 9

Reply:
I'd think [here](https://github.com/davidslater/armory/blob/mot-coco/armory/data/adversarial_datasets.py#L1115) is the place the class should be referenced

Reply:
Good catch on all of the above, will change.

Reply:
It indexes the first of the 9 because that's the frame_id field, so it's using it to create an index.